wp-content/civi-extensions/goonjcustom/Civi/InstitutionService.php

```diff
@@ -173,7 +174,7 @@ private static function getFallbackCoordinator($contactId) {
    *
    */
   private static function getRelationshipTypeName($contactId) {
-    $organization = Organization::get(TRUE)
+    $organization = Organization::get(FALSE)
       ->addSelect('Institute_Registration.Type_of_Institution:label')
       ->addWhere('id', '=', $contactId)
       ->execute()->single();
```

Comment:
If contact not exist then this query will give error do you have tested that case?

Reply:
It never breaks because I've added a check before this condition to return if no contact ID is found:
```
if (!$contactId) 
{
 \CRM_Core_Error::debug_log_message('Contact ID not found'); 
 return; }
 ```